planner_cspace/src/planner_3d.cpp

```diff
@@ -1046,6 +1058,13 @@ class Planner3dNode
       act_->setAborted(move_base_msgs::MoveBaseResult(), "Given goal is invalid.");
   }
 
+  void cbTolerantAction()
+  {
```

Comment:
`MoveWithTolerance` and `MoveBase` actions should be run exclusively.

Reply:
@at-wat Done.